.gitignore

```diff
@@ -40,3 +40,6 @@ hs_err_pid*
 # Ignore jekyll build output.
 /production
 /.sass-cache
+
+# Ignore mac attributes files
+*.DS_STORE
```

Comment:
this is unrelated to the subject of the PR; how strict is it in OSS to require separate PRs for separate things?